setup.py

```diff
@@ -21,20 +21,20 @@
 LICENSE = 'MIT'
 AUTHOR = 'Rdtools Python Developers'
 MAINTAINER_EMAIL = 'adam.b.shinn@gmail.com'
-URL = 'https://github.com/kwhanalytics/rdtools'
+URL = 'https://github.com/NREL/rdtools'
 
 INSTALL_REQUIRES = [
     'numpy >= 1.11.0',
     'pandas >= 0.19.0',
     'pvlib >= 0.5.0',
     'statsmodels >= 0.8.0',
     'patsy >= 0.4.1',
-    'nose >= 1.3.7',
+    'h5py >= 2.7.1',
     'pytz',
```

Comment:
Let's add `'scipy >= 0.19.1'`. I think it will generally be installed with `statsmodels`, but we use it explicitly in `normalization.py` so probably best to specify it.